### PR TITLE
Added Graph Fundamentals Resources

### PR DESCRIPTION
**Pull request by PritikaSabharwal**
Added hackerEarth graph tutorials under graph theory

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,7 +194,8 @@ To contribute, please check out our contribution guide [here](https://github.com
 ### Graph Theory
 
 - [Introduction to Graph Theory](https://www.udemy.com/course/graph-theory-algorithms/learn/)
-
+- [Graph Fundamentals Part-1](https://www.hackerearth.com/practice/notes/graph-theory-part-i/)
+- [Graph Fundamentals Part-2](https://www.hackerearth.com/practice/notes/graph-theory-part-ii/)
```
Comment (shahednasser) [inline README.md:198]: Please add a new line under this.

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Review by shahednasser — CHANGES_REQUESTED** [review]
I think you misunderstood. I meant add a new line like a space between the items and the horizontal line below the "Graph Theory" list. You don't need to add it in GIT

**Review by PritikaSabharwal — COMMENTED** [review]
Sir is this correct ?

File: README.md [hunk 2]
```diff
@@ -218,6 +220,8 @@ To contribute, please check out our contribution guide [here](https://github.com
 
 - [GraphQL tutorial for beginners](https://www.howtographql.com/)
 - [React And GraphQL](https://medium.com/codingthesmartway-com-blog/getting-started-with-react-and-graphql-395311c1e8da)
+- [Graph Fundamentals Part-1](https://www.hackerearth.com/practice/notes/graph-theory-part-i/)
```
Comment (shahednasser) [inline README.md:223]: Remove those two resources as they don't relate to GraphQL

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Pull request closed without merging**